tests/data/expected/otr/test_1.otr

```diff
@@ -0,0 +1 @@
+{"text":"        <p><span class=\"timestamp\" data-timestamp=\"9.483782\">00:09</span> ང་ཚོ་རྒྱ་གར་དུ་སླེབས་ནས་རིམ་པའི་གྱི་ ཅིག ཨའ།</p><p><br /><span class=\"timestamp\" data-timestamp=\"17.449519\">00:17</span>  དགོན་པ་དང་།  ལྷག་པར་དུ་ ཨ་ནི། བཙུན་མའི་དགོན་པ་ཚོའི། ད་དེ་སྔ་དགོན་པ་རིགས་ཁ་ཤས་ཀྱི་</p><p><br /><span class=\"timestamp\" data-timestamp=\"26.632803\">00:26</span> སྔགས་ཆོག་གདོན་སྡོད་པ་མ་གཏོགས། འ་འ། ཅིག་དེ་འདྲ་ཡི་ཕར་གི་རིགས་ལམ་</p><p><br /><span class=\"timestamp\" data-timestamp=\"35.100599\">00:35</span> སྦྱངས་ནས་གཞུང་ཆེན་བལྟ་ཡ་  འ་ མེད་མཁན་དེ་འདྲ་མང་པོ་ཞིག་ཡོད་པ་རེད། བཙུན་མའི་དགོན་པ་ཚོ་ནི་ཧ་ལམ་ད་</p><p><br /><span class=\"timestamp\" data-timestamp=\"44.148338\">00:44</span> དེ་འདྲའི་གཞུང་ཆེན་བལྟས། རིགས་ལམ་སྦྱོང་། ཧ་ལམ་བྱེད་མཁན། བྱེད་མཁན་ཧ་ལམ་ཡོད་མ་རེད་ད། བྱས་ཙང་འདི་</p><p><br /><span class=\"timestamp\" data-timestamp=\"52.986499\">00:52</span> ང་རང་ཚོས་དེ་སྔ་ཕན་བྱེད་ཀྱི་གོམས་གཤིས་ཤིག་རེད་པས་ཟེར་ལབ་ན། གོམས་གཤིས་ཤིག་རེད། བཙུན་མ་དགོན་པོ་ཚོ་རྟགས་གསལ་གཏོང་ཡ་ལུགས་སྲོལ་མེད་སྡོད་པ་ཞིག་རེད།</p><p><br /><span class=\"timestamp\" data-timestamp=\"61.122624\">01:01</span>  སྔགས་པ་གྲྭ་ཚང་ཟེར་ཡ་དེ་ཚོ་དེ་འདྲ་ལོ་བརྒྱ་ཕྲག་རིང་ལ་ཡང་འདོན་ཆོག་རང་བྱས་ནས་འདོན་བསྡད་པ་མ་གཏོགས་ཡོད་མ་རེད། བྱས་ཙང་ད་འདི་</p><p><br /><span class=\"timestamp\" data-timestamp=\"71.397616\">01:11</span> དེ་སྔས་ཁོ་རང་ལུགས་སྲོལ་འདྲ་བོ་ཁོ་རང་བཞག་དགོས་རེད་ལབ་ན། </p><p><br /><span class=\"timestamp\" data-timestamp=\"78.053466\">01:18</span> སྔོན་གྱི་ལུགས་སྲོལ་ཡིན་པ་ཡིན་ན་ཡང་ངོ་བོ་ར་བོ་ཁོ་ཕན་ཐོགས་ཡོད་པ་ཞིག་རེད་འདུག་</p><p>རྗེས་ལུས་རེད་འདུག་</p><p><br /><span class=\"timestamp\" data-timestamp=\"86.883668\">01:26</span> བསམ་བློ་གཏོང་དགོས་རེད་པ། བྱས་ཙང་དེ་འདྲ་ཡིན་དུས། འདུག་སའི་བསམ་བློ་ཞིག་གཏོང་པ་ཡིན་ན་དེ་རྗེས་ལུས་རེད་མ་གཏོགས། </p><p><br /><span class=\"timestamp\" data-timestamp=\"95.606826\">01:35</span> ཁོ་ཕི་གི་ཡོན་ཏན་ལེགས་ཆ་ཡོད་སྡད་པ་ཞིག་གལ་ཡོད་མ་རེད། བྱས་ཙང་དེ་འདྲ་ཡིན་ཙང་ང་ཚོ་རྒྱ་གར་ཡ་སླེབས་ནས། ཨ་ནི།</p><p><br /><span class=\"timestamp\" data-timestamp=\"103.634127\">01:43</span>  ད་ག་ཕན་བདེ་ལེགས་བཤད་གླིང་གི་མཚོན་པའི་རྒྱུད་གྲྭ་སྨད་སྟོད་རེད། ཅིག་དེ་ཚོ་ལ་ཡ།</p><p><br /><span class=\"timestamp\" data-timestamp=\"111.874609\">01:51</span> དེ་སྔ་ཨ་ནི། བླ་མ་རྒྱུད་པ་ལ་ཆ་བཞག་པ་ཡིན་ན་བསྐྱེད་རིམ་པའི་གྲས་འདི་གས་འདི་མཚན་ཉིད་ལྟ་གི་ཡོད་མ་རེད། མཚན་ཉིད་ལྟ་དགོས་རེད། མཚན་ཉིད་བཙུགས་པ་རེད།</p><p><br /><span class=\"timestamp\" data-timestamp=\"121.609916\">02:01</span>དེ་ཙམ་དུ་མ་ཟད་པའི་བཙུན་མའི་ཚོ་ལའི་དགེ་སློང་མ་བསྒྲུབ་</p><p><br /><span class=\"timestamp\" data-timestamp=\"129.411759\">02:09</span> ཡ་ཟེར་མཁན་འདི་ཡང་ཁོ་རང་འདུལ་བའི་གཞུང་དང་བསྟུན་ནས་འགྲོ་དགོས་པ་རེད་དོ། །</p><p><br /><span class=\"timestamp\" data-timestamp=\"132.713335\">02:12</span>  </p><p><br /></p><p><br /></p><p><br /></p><p>    </p>","media":"","media-source":"","media-time":""}
```

Comment:
in case of test folder structure i would recommand [toolkit test](https://github.com/OpenPecha/Toolkit/tree/master/tests)

keep file name as expected_otr.otr rather then test_1.otr. Naming recommendation applies to all of the test files.

Reply:
done